changes/34ba15e2aef8b5c6be4d0274db884f84.yaml

```diff
@@ -0,0 +1,5 @@
+---
+desc: Refactored backup streaming logic, error handling, and increased test coverage.
```

Comment:
Don't need to include test coverage in the changelog entry IMO.
```suggestion
desc: Refactored backup streaming logic and error handling.
```

Reply:
Thanks!